src/SideNav.tsx

```diff
@@ -97,6 +103,13 @@ const SideNavLink = styled(Link).attrs<StyledSideNavLinkProps>(props => {
 
     &:first-child {
       border-top: 0;
+      border-top-right-radius: ${get('radii.2')};
+      border-top-left-radius: ${get('radii.2')};
```

Comment:
I've move this down from  lines 82 - 91 due do to the following:

![Screen Shot 2021-05-13 at 8 46 55 PM](https://user-images.githubusercontent.com/6519974/118316610-e5a2ef80-b4ab-11eb-9208-abbef61eb7e3.png)

for the lightweight version